### ISSUE TITLE
Tacere doesn't fetch any calendar

### ISSUE THREAD

**Issue by Captain-Rage** [ciasaboark/Tacere#46]
My GoClever Quantum 4 is running Android 4.2.2, which I set up to use DAVdroid. I installed Tacere 2.04 from the F-Droid repository yesterday. Tacere starts but it doesn't want to import any calendar. It only says "No calendar events found for the next 1 week."

**Comment by ciasaboark** [comment]
Can you confirm that DAVdroid is syncing calendars correctly?  Do calendars and events show up in the google calendar app (or any other third party calendar app that uses the system calendar database)?  I don't have any experience using DAVdroid, but if events are not visible in other calendar apps then you might want to check https://davdroid.bitfire.at/faq/entry/entries-dont-get-synchronized

I haven't maintained the F-Droid variant very well.  I would recommend upgrading to a newer build.  You can find a direct link to the latest stable apk at http://phobotic.io/projects/tacere/

The latest release allows individual calendar selection, and fixed a number of bugs from the 2.0 base.  It also uses in-app upgrades to enable additional functionality (which is why the f-droid variant is behind).  In order to select calendars you will either need to purchase the in-app upgrade in the google play store, or install the older donate key https://github.com/ciasaboark/Tacere/blob/master/release/Tacere-Donate.apk

Once you have upgraded check that calendars are visible in settings->select calendars

**Comment by Captain-Rage** [comment]
Yes, indeed the calendar entries show up in Google Calendar (they can be modified and synced to other devices as well, using Radicale as server). They simply don't show up (get imported?) to Tacere 2.1.0 beta 5. Clicking on "Add new event" brings me to Google Calendar window for creating a new event.
I tried checking and unchecking "Sync All Calendars" but the same behaviour remains.

**Comment by Captain-Rage** [comment]
I deem Tacere 2.1.0 beta 5 hopeless on my phone. Whatever I seem to do it won't import any calendar, only displaying "No calendar events found".

After installing tacere.v2.1.2.apk and Tacere-Donate.apk the behaviour changed for the better. First it still wouldn't fetch any calendar events. After adding adding an event in Google Calendar through Tacere two events would show up (the one just created and the next one) but no other for that day. Finally, after changing the "Date Range" to "Three Months" every event for the intended calendar appeared.

Tacere 2.1.2 seems to play along nicely now, but on initial startup (as well as for Date Range: One Week) it refuses to fetch any calendar events.

Tell me if I can help you to debug this issue. Thanks for the neat app and technical support!
